posts/2022-07-05-logrecordcontext-22007.adoc

```diff
@@ -0,0 +1,339 @@
+---
+layout: post
+title: "Enhancements to LogRecordContext extension and improved stack traces in Open Liberty 22.0.0.7"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Enhancements to LogRecordContext extension and improved stack traces in Open Liberty 22.0.0.7 - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.7 includes the addition of the application name to the LogRecordContext extension and a corresponding JSON logging field, as well as providing new logging functionality to join stack traces in logs so that they are output as a single log event. Also included in this release are several notable bug fixes and updates to our Open Liberty guides.
```

Comment:
```suggestion
seo-description: Open Liberty 22.0.0.7 includes the addition of the application name to the LogRecordContext extension and a corresponding JSON logging field, as well as providing new logging functionality to join stack traces in logs so that they are output as a single log event and easier to consume. Also included in this release are several notable bug fixes and updates to our Open Liberty guides.
```